src/main/java/io/jenkins/plugins/designlibrary/Root.java

```diff
@@ -24,6 +31,22 @@ public String getUrlName() {
         return "design-library";
     }
 
+
+    @Override
+    public ContextMenu doContextMenu(StaplerRequest request, StaplerResponse response) throws Exception {
```

Comment:
I tried just using
`return new ContextMenu().from(this,request,response);`

But this didn't work, the menu is empty.